<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tutorialSQLITE</class>
 <widget class="QDialog" name="tutorialSQLITE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_conectar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>desconectar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_desconectar">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>executar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_executar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>conectar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_schema">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>schema</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_schema">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>20</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_sql">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>571</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tbl_sql">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>571</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
